arangopipe/Dockerfile_Prod

```diff
@@ -0,0 +1,8 @@
+FROM python:3.7
+RUN pip install jsonpickle python-arango jupyter sklearn2 PyYAML==5.1.1
+RUN pip install -i https://test.pypi.org/simple/ arangopipe
```

Comment:
For prod I would prefer not to pull from test and fix the all the package versions

Reply:
sure. I just wanted to get feedback on the approach before migrating the package to PyPi. Will push arangopipe to PyPi.

Reply:
Go ahead

Reply:
@joerg84 , put the latest arangopipe build into PyPi and updated docker files to use PyPi packages.